poi/src/main/java/org/apache/poi/hssf/model/InternalSheet.java

```diff
@@ -541,12 +540,11 @@ public ConditionalFormattingTable getConditionalFormattingTable() {
     public void setDimensions(int firstrow, short firstcol, int lastrow, short lastcol)
     {
         LOGGER.atDebug().log("Sheet.setDimensions");
-        LOGGER.atDebug().log(() -> new SimpleMessage(
```

Comment:
Correct. And while we are at it: since we are at Log4J2 now, I think we should instead better use something like this:
```
        LOGGER.debug("Sheet.setDimensions");
        LOGGER.debug("firstrow{}firstcol{}lastrow{}lastcol{}", firstrow, firstcol, lastrow, lastcol);

```

So, get rid of the `atDebug()`where not necessary and use the MessageFormat.

Reply:
if we do change this line, I suggest that we improve the format of its output - eg 
```
LOGGER.debug("firstrow: {} firstcol: {} lastrow: {} lastcol: {}", firstrow, firstcol, lastrow, lastcol);
```